### PR TITLE
Add commas and ands to in partnership with list.

### LINKED ISSUE #1457: action page (partners) - comma and "and" logic with multiple partners
If there are multiple partners attached to a campaign we should have commas and the word "and" separating them.

So if there are two partners, we just need "and" separating them. If there are three or more partners we'd need commas separating each and "and" right before the last comma.

example: https://beta.dosomething.org/campaigns/cell-phones-survivors

image

### PR DESCRIPTION
**Pull request by angaither**
Moved to module
Fixes #1457

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php [hunk 1]
```diff
@@ -394,6 +394,9 @@
             In partnership with
             <?php foreach ($partners as $key => $partner) :?>
               <?php print $partner['name']; ?>
+              <?php if (count($partners) > 2 && $key == (count($partners) -2)): print 'and '; ?>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php]: Should it be `' and '` ?

Comment (angaither) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php]: It actually works fine without that, there's spaces before the commas, even though I didn't add them. @DFurnes @weerd do you know if extra spaces are being added?

**Comment by weerd** [comment]
Nothing's going on CSS-wise that would be affecting it. I suggested to @angaither to run the output through trim() before printing to try and remove extra whitespace so that it can me more consistent :)

**Comment by angaither** [comment]
@aaronschachter updated!

**Comment by aaronschachter** [comment]
Looks great :+1:

**Pull request merged**